src/components/m-main-navigation-mobile/index.scss

```diff
@@ -18,14 +38,36 @@
   background: $main-navigation-mobile-bg;
 }
 
-.m-main-navigation-mobile--off-canvas {
-  position: fixed;
+.m-main-navigation-mobile__canvas--off-canvas {
+  @include hardware-accelerate();
 
-  @include respond-up(md) {
+  position: absolute;
+  top: 100%;
+  right: 0;
+  z-index: $main-navigation-mobile__nav-z-index;
+
+  height: 100%;
+  margin-top: -1px;
+  overflow-x: hidden;
+  overflow-y: scroll;
+  -webkit-overflow-scrolling: touch;
```

Comment:
wir brauchen noch den autoprefixer

Reply:
Absolutely, but I guess `-webkit-overflow-scrolling` is webkit only